tasks/windows.ps1

```diff
@@ -0,0 +1,82 @@
+[CmdletBinding()]
+param(
+  # NOTE: init.json cannot yet be shared, so must have windows.json / windows.ps1
+  [Parameter(Mandatory = $true)]
+  [String]
+  $Name,
+
+  [Parameter(Mandatory = $true)]
+  [ValidateSet('start', 'stop', 'restart', 'status')]
+  [String]
+  $Action
+)
+
+$ErrorActionPreference = 'Stop'
+
+function Invoke-ServiceAction($Service, $Action)
+{
+  $inSyncStatus = 'in_sync'
+  $status = $null
+
+  switch ($Action)
+  {
+    'start'
+    {
+      if ($Service.Status -eq 'Running') { $status = $InSyncStatus }
+      else { Start-Service $Service }
+    }
+    'stop'
+    {
+      if ($Service.Status -eq 'Stopped') { $status = $InSyncStatus }
+      else { Stop-Service $Service }
+    }
+    'restart'
+    {
+      Restart-Service $Service
+      $status = 'restarted'
+    }
+    # no-op since status always returned
+    'status' { }
+  }
+
+  # user action
+  if ($status -eq $null)
+  {
+    # https://msdn.microsoft.com/en-us/library/system.serviceprocess.servicecontrollerstatus(v=vs.110).aspx
+    # ContinuePending, Paused, PausePending, Running, StartPending, Stopped, StopPending
+    $status = $Service.Status
+    if ($status -eq 'Running') { $status = 'started' }
+  }
+
+  return $status
+}
+
+try
+{
+  $service = Get-Service -Name $Name
+  $status = Invoke-ServiceAction -Service $service -Action $action
+
+  # TODO: could use ConvertTo-Json, but that requires PS3
+  # if embedding in literal, should make sure Name / Status doesn't need escaping
+  Write-Host @"
+  {
+    "Name"        : "$($service.Name)",
+    "DisplayName" : "$($service.DisplayName)",
+    "Status"      : "$status",
+    "StartType"   : "$($service.StartType)"
+  }
+"@
+}
+catch
+{
+  Write-Host @"
+  {
+    "status"  : "failure",
+    "error"  : {
```

Comment:
The original task uses `_error`, but Bolt is unable to properly parse the return value with the `_` present in PowerShell

For instance, given  `Write-Host '{ "error" : true }'` the response is:

```
Started on 127.0.0.1...
Finished on 127.0.0.1:
  {
    "error": true
  }
Successful on 1 node: winrm://127.0.0.1:55985
Ran on 1 node in 4.77 seconds
```

However, changing to `Write-Host '{ "_error" : true }'`, Bolt fails like

```
Started on 127.0.0.1...
Failed on 127.0.0.1:
Failed on 1 node: winrm://127.0.0.1:55985
Ran on 1 node in 4.71 seconds
```

/cc @MikaelSmith  @nicklewis

Reply:
@MikaelSmith says what I'm seeing is by design, though it was a little surprising to me.

I have changed it back to `_error`.